versions.yaml

```diff
@@ -329,14 +329,14 @@ externals:
   virtiofsd:
     description: "vhost-user virtio-fs device backend written in Rust"
     url: "https://gitlab.com/virtio-fs/virtiofsd"
-    version: "v1.6.1"
+    version: "v1.8.0"
     toolchain: "1.66.0"
```

Comment:
@skaegi, mind to also bump the toolchain version to 1.72.0?

The reason for that is that virtiofsd has been using the alpine:rust image (see: https://gitlab.com/virtio-fs/virtiofsd/-/blob/main/.gitlab-ci.yml?ref_type=heads#L6), which right now used the 1.72.0 version of the toolchain.

Reply:
Sure - done!